os/board/common/partitions.c

```diff
@@ -114,7 +114,7 @@ static int type_specific_initialize(FAR struct mtd_dev_s *mtd_part, int partno,
 	}
 #endif
 #ifdef CONFIG_LIBC_ZONEINFO_ROMFS
-	else if (!strncmp(types, "timezone,", 10)) {
+	else if (!strncmp(types, "timezone,", 9)) {
```

Comment:
This seems valid.

Reply:
"timezone,a" will be return true. Is it ok?